### PR TITLE
Adds intermediate NYC taxis tutorial

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds a new intermediate tutorial using the NYC taxis dataset. Moves the advanced queries section from the original beginner tutorial into the intermediate, and adds in the geospatial mapping tutorial from the old Grafana tutorials in as well. Plus editing, new screenshots, etc 😉 

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by Rajakavitha1 — APPROVED** [review]
LGTM! Thank you for this tutorial. I was struggling with the energy tutorial partials and trying to maximise reuse. This tutorial helped me a lot and I shamelessly copied your approach.

File: tutorials/nyc-taxi-geospatial/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+1.  [Setting up up your dataset][dataset-nyc]: Set up and connect to a Timescale
+    service, and load data into your database using `psql`.
+1.  [Querying your dataset][query-nyc]: Analyze a dataset containing NYC taxi
+    trip data using Timescale and PostgreSQL, and plot the results in Grafana.
+
+## About querying data with Timescale
+
+This tutorial uses the [NYC taxi data][nyc-tlc] to show you how to construct
+queries for geospatial time-series data. The analysis you do in this tutorial is
+similar to the kind of analysis civic organizations use to do things like plan
```
Comment (Rajakavitha1) [inline tutorials/nyc-taxi-geospatial/index.md:48]:
```suggestion
similar to the kind of analysis civic organizations do to plan
```

File: tutorials/nyc-taxi-geospatial/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+new roads and public services.
+
+It starts by teaching you how to set up and connect to a Timescale database,
+create tables, and load data into the tables using `psql`. If you have already
+completed the [beginner NYC taxis tutorial][beginner-fleet], then you already
+have the dataset loaded, and you can skip straight to the queries.
```
Comment (Rajakavitha1) [inline tutorials/nyc-taxi-geospatial/index.md:54]: Wondering if we need a link to the queries section.

File: tutorials/nyc-taxi-geospatial/plot-nyc.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+     2016-01-01 01:00:00 |           120
+     2016-01-01 01:30:00 |            98
+     2016-01-01 02:00:00 |           112
+    ```
+
+</Procedure>
+
+## Which rides traveled more than 5 miles in Manhattan?
+
+This query is especially well suited to being plotted on a map. It looks at
```
Comment (Rajakavitha1) [inline tutorials/nyc-taxi-geospatial/plot-nyc.md:111]:
```suggestion
This query is especially well suited to plot on a map. It looks at
```

**Review by Rajakavitha1 — APPROVED** [review]

File: tutorials/nyc-taxi-geospatial/index.md (hunk 2 above)
Comment (Loquacity) [reply tutorials/nyc-taxi-geospatial/index.md:54]: Yeah, I can do that 👍

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!!! Hi @Loquacity This is ready for merge. During the review I noticed that the Installing psql docs has the instructions repeated and I will address in a new PR.

**Pull request merged**